using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -680,16 +778,19 @@ If the current SDK privacy status is `optedout`, the advertising identifier is n
 
 {% tabs %}
 {% tab title="Android" %}
-#### Java
 
-### setAdvertisingIdentifier
+### setAdvertisingIdentifier<a id="setAdvertisingIdentifier-java"></a>
+
+This `setAdvertisingIdentifier()` API sets  the advertising identifier provided. 
```

Comment:
- Please remove `setAdvertisingIdentifier()`.
- Please move "provided" to after "the". 
- Remove the extra space between "sets" and "the".